packages/core/index.js

```diff
@@ -4,7 +4,6 @@ var pkg = require("./package.json");
 module.exports = {
   build: require("./lib/build"),
   create: require("./lib/commands/create/helpers"),
-  console: require("./lib/repl"),
```

Comment:
So this would be a breaking change for `@truffle/core`, right?
Maybe you should keep the legacy stuff in and just file it in a `legacy` folder so it won't be?

Reply:
Hm, I guess it's possible someone was using the ReplManager externally? It was really an internal concern though, and I don't think it ever should have been exposed. Wondering what @gnidan thinks - is this a breaking change and should I do something to keep it from being such?